doc/runtime-information.rst

```diff
@@ -59,12 +59,70 @@ bundled and in the bundle folder if it is bundled::
     bundle_dir = getattr(sys, '_MEIPASS', path.abspath(path.dirname(__file__)))
     path_to_dat = path.abspath(path.join(bundle_dir, 'other-file.dat'))
 
+Or, if you'd rather use pathlib_::
+
+    from pathlib import Path
+    import sys
+
+    if getattr(sys, 'frozen', False) and hasattr(sys, '_MEIPASS'):
+        bundle_dir = Path(sys._MEIPASS)
+    else:
+        bundle_dir = Path(__file__).parent
+
+    path_to_dat = Path.cwd() / bundle_dir / "other-file.dat"
```

Comment:
Isn't this still relative? And why can't you use `bundle_dir / "other-file.dat"`?

Reply:
No, `Path.cwd() / bundle_dir` is equivalent to `bundle_dir.absolute()`. See [the link](https://discuss.python.org/t/pathlib-absolute-vs-resolve/2573/15) and @BoboTiG's review. 

Although perhaps we should scrap this *always use absolute paths* as the case for relative paths causing issues is very rare.

Reply:
Just recommend they use absolute paths. Otherwise 👍